### PR TITLE
Fix select all button bug

### PR DESCRIPTION
**Pull request by eldadfux**
## What does this PR do?

Fixed bug where select/unselect all button had weird behavior.

## Test Plan

N/A

## Related PRs and Issues

#477 

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

File: src/Appwrite/Utopia/Response.php [hunk 1]
```diff
@@ -257,8 +258,7 @@ public function dynamic(Document $document, string $model): void
             $item = self::getFilter()->parse($output, $model);
         }
 
-        $this->json($output);
-
+        $this->json(!empty($output) ? $output : new stdClass());
```
Comment (christyjacob4) [inline src/Appwrite/Utopia/Response.php:261]: Can we not do the following here 
```suggestion
        $this->json($output ?? new stdClass());
```

**Review by christyjacob4 — CHANGES_REQUESTED** [review]
Rest Looks good @eldadfux

File: src/Appwrite/Utopia/Response.php (hunk 1 above)
Comment (eldadfux) [reply src/Appwrite/Utopia/Response.php:261]: I wasn't 100% sure so I doubled check, and the output is not the same for both lines. ?? is similar to `isset` and check if the assoc array has a specific key. Empty will also return true for empty string or arrays.

This is the code I used for the test:

```php
$output = [];

echo "\n";

echo '1:'.json_encode(!empty($output) ? $output : new \stdClass());

echo "\n";

echo '2:'.json_encode($output ?? new \stdClass());
```

results:
```php
// 1:{}
// 2:[]
```

Comment (christyjacob4) [reply src/Appwrite/Utopia/Response.php:261]: okay eldad 👍

**Pull request merged**